benchmarkapps/BasicApi/Startup.cs

```diff
@@ -68,7 +68,7 @@ public void ConfigureServices(IServiceCollection services)
                 case "MYSQL":
```

Comment:
Is the #if above needed? I cannot think of a reason from the provider or EF Core side.

Reply:
It's necessary because this project is also built targeting .NET 4.6.1. The lack of a strong name on this EF provider simply breaks builds in that case. So, I simply removed the option.